package.json

```diff
@@ -20,14 +20,14 @@
     "build:css": "NODE_ENV=production postcss src/tailwind.css -o src/index.css",
     "build:js": "NODE_ENV=production react-app-rewired build",
     "eject": "react-scripts eject",
-    "blockchain": "ganache-cli --mnemonic \"indicate swing place chair flight used hammer soon photo region volume shuffle\"",
-    "integration": "./scripts/integration.sh",
-    "integration:single": "./scripts/integration-single.sh",
-    "integration:headless": "HEADLESS=true ./scripts/integration.sh",
-    "integration:internal": "testcafe chrome integration/**/*.spec.ts --app \"BROWSER=none npm run serve-static\"",
-    "integration:internal:single": "testcafe chrome -L --app \"npm run serve-static\"",
-    "integration:internal:headless": "testcafe -c 4 chrome:headless integration/**/*.spec.ts --app \"npm run serve-static\" --app-init-delay 70000",
-    "integration:concurrently:headless": "concurrently -k -s first \"npm:blockchain\" \"npm:integration:headless\"",
+    "blockchain": "run-p blockchain:start blockchain:setup",
+    "blockchain:contracts": "node ./scripts/setup-contracts.js",
+    "blockchain:setup": "run-s wait:blockchain blockchain:contracts",
+    "blockchain:start": "ganache-cli --mnemonic \"indicate swing place chair flight used hammer soon photo region volume shuffle\"",
+    "integration": "testcafe chrome integration/**/*.spec.ts --app \"npm run serve-static\"",
+    "integration:ci": "concurrently -k -s first \"npm:blockchain\" \"npm:integration:headless\"",
+    "integration:headless": "testcafe -c 3 chrome:headless integration/**/*.spec.ts --app \"npm run serve-static\"",
```

Comment:
why 3? no particular reason copied from testcafe docs. hopefully it helps with flakiness